app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -0,0 +1,91 @@
+class ManageIQ::Providers::Vmware::CloudManager < ManageIQ::Providers::CloudManager
+  def self.ems_type
+    @ems_type ||= "vmware_cloud".freeze
+  end
+
+  def self.description
+    @description ||= "VMware vCloud".freeze
+  end
+
+  def self.default_blacklisted_event_names
+    []
+  end
+
+  def self.hostname_required?
+    true
+  end
+
+  def supports_port?
+    true
+  end
+
+  def supported_auth_types
+    %w(default)
+  end
+
+  def supports_authentication?(authtype)
+    supported_auth_types.include?(authtype.to_s)
+  end
+
+  #
+  # Connections
+  #
+
+  def self.raw_connect(server, port, username, password)
+    require 'fog/vcloud_director'
+
+    params = {
+      :vcloud_director_username      => username,
+      :vcloud_director_password      => password,
+      :vcloud_director_host          => server,
+      :vcloud_director_show_progress => false,
+      :port                          => port,
+      :connection_options            => {
+        :ssl_verify_peer => false # for development
+      }
+    }
+
+    Fog::Compute::VcloudDirector.new(params)
+  end
+
+  def connect(options = {})
+    raise "no credentials defined" if missing_credentials?(options[:auth_type])
+
+    server   = options[:ip] || address
+    port     = options[:port] || self.port
+    username = options[:user] || authentication_userid(options[:auth_type])
+    password = options[:pass] || authentication_password(options[:auth_type])
+
+    self.class.raw_connect(server, port, username, password)
+  end
+
+  def vcd_service
+    @vcd_service ||= connect
+  end
```

Comment:
I find this strange, as the pattern is typically to have a method called `with_provider_connection`.  @blomquisg @agrare  Thoughts here?